### PR TITLE
Remove one pane

### PR DESCRIPTION
**Pull request by garrensmith**
Remove the one_panel.html and use react instead

### CODE REVIEW

File: bin/create-animal-db [hunk 1]
```diff
@@ -16,7 +16,7 @@
 // with conflicts for the zebra doc
 
 
-const url = 'http://localhost:5984/';
+const url = 'http://tester:testerpass@localhost:5984/';
```
Comment (robertkowalski) [inline bin/create-animal-db]: debug issue?

**Comment by robertkowalski** [comment]
+1, merge it after removing the user/pass from the script :) 👍

File: bin/create-animal-db (hunk 1 above)
Comment (garrensmith) [inline bin/create-animal-db]: I would like to change it to that. We always end up using the docker couchdb instance anyway. This is so much easier

**Comment by garrensmith** [comment]
merging since this is green https://travis-ci.org/garrensmith/couchdb-fauxton/builds/177983807

**Pull request closed without merging**